amd/src/mutations.js

```diff
@@ -347,4 +347,13 @@ export default class {
         let updates = JSON.parse(result.update);
         stateManager.processUpdates(updates);
     }
+
+    /**
+     * Duplicate a card.
+     * @param {*} stateManager
```

Comment:
`* @param {*} stateManager StateManager instance.`
`* @param {number} cardId Id of the card to duplicate.`